zenodo/modules/records/templates/zenodo_records/box/info.html

```diff
@@ -96,22 +96,24 @@
   {%- if i and not p %}
     <dt>{{_('Imprint')}}:</dt>
     <dd>
-      {{i.publisher}}{% if i.place %}, {{i.place}}{%- endif %}{% if i.isbn %} ({{i.isbn}}){%- endif %}.
-    </dd>
-  {%- elif p %}
-    <dt>{{_('Published in')}}:</dt>
-    <dd>
-      {{p.title}}{% if i.place %}, {{i.publisher}}{%- endif %}{% if i.place %}, {{i.place}}{%- endif %}{% if p.pages %}, pp. {{p.pages}}{%- endif %}{% if i.isbn %} ({{i.isbn}}){%- endif %}.
+      {{i.publisher}}{% if i.place %}, {{i.place}}{%- endif %}{% if i.isbn %}({{i.isbn}}){%- endif %}.
     </dd>
+  {%- elif p.title %}
```

Comment:
Merging of the two conditions from here: https://github.com/zenodo/zenodo/pull/981#discussion_r104384130

should result in ``{%- elif p and p.title %}``. This part will raise an exception if ``p`` is None.